plugins/basic/nexus-ui-extjs3-plugin/src/main/resources/static/js/Sonatype/utils.js

```diff
@@ -818,7 +817,6 @@ define('Sonatype/utils',['../extjs', 'Nexus/config', 'Nexus/util/Format', 'Sonat
                   method : 'GET',
                   url : Sonatype.config.repos.urls.logout,
                   callback : function(options, success, response) {
-                    ns.clearCookie('JSESSIONID');
```

Comment:
If login fails ask to logout? again, makes no sense

removing this JS helps avoid passing the custom cookie name back to the UI in a header or something as well